cmd/vic-machine/configure/configure.go

```diff
@@ -184,6 +257,10 @@ func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n
 			v.Network.Nameservers = n.ExecutorConfig.Networks[k].Network.Nameservers
 		}
 	}
+	o.HostCertificate = n.HostCertificate
```

Comment:
hmm... I think this is the last step to change VCH configuration, if you override the existing value with a nil, it will be persisted.